uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -257,14 +259,15 @@ async def send(self, message):
                 extensions = []
                 if self.config.ws_per_message_deflate:
                     extensions.append(PerMessageDeflate())
-                output = self.conn.send(
-                    wsproto.events.AcceptConnection(
-                        subprotocol=subprotocol,
-                        extensions=extensions,
-                        extra_headers=extra_headers,
+                if not self.transport.is_closing():
+                    output = self.conn.send(
+                        wsproto.events.AcceptConnection(
+                            subprotocol=subprotocol,
+                            extensions=extensions,
+                            extra_headers=extra_headers,
+                        )
                     )
-                )
-                self.transport.write(output)
+                    self.transport.write(output)
```

Comment:
This is necessary on this scenario, because the transport was closed on the `shutdown()`, but the server still didn't accept the connection, but... Well... It's too late to accept it. Sorry! :eyes: